.eslintrc.json

```diff
@@ -23,6 +23,7 @@
     "import/no-extraneous-dependencies": ["error", { "devDependencies": ["**/*.test.*", "./*"] }],
     "jsx-a11y/label-has-associated-control": ["error", { "assert": "either" }],
     "no-shadow": "off",
+    "no-unused-vars": "off",
```

Comment:
without this, eslint complained about "unused var Step" ... which is most definitely used, there's a `@typescript/no-unused-vars` we already have enabled as the workaround